Gemfile

```diff
@@ -3,7 +3,11 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.7.0'
 
+gem 'bootstrap', '~>4.0.0'
+gem 'jquery-rails'
+
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'kaminari'
```

Comment:
What does gem `kaminari` do?

Reply:
Gem Kaminari allows performing paging in a fast and clean way in the code, implementing views, that permits the realization of different jumps between the paging of the models to which they are implemented.